### PR TITLE
Profile picture from Google Account

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
- Add Using the profile picture from Google Account

### CODE REVIEW

**Comment by KAYEMBAJONATHAN** [comment]
Screenshot 2024-08-03 180510

File: src/app/components/hris/nav-bar/types/top-nav/top-nav.component.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export class TopNavComponent {
   ) {
     this.screenWidth = window.innerWidth;
     this.dialogTypeData = new DialogTypeData().dialogTypeData;
+    var test = authAccessService.getAuthTokenProfilePicture();
```
Comment (bseptember) [inline src/app/components/hris/nav-bar/types/top-nav/top-nav.component.ts:55]: Refactor: remove this line

File: src/app/components/hris/sign-in/sign-in.component.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ import { selectToken } from '../../shared-components/store/selector/sign-in.sele
 export class SignInComponent {
   user: Token | undefined;
   userEmail: string | null = null;
+  photoUrl: string | undefined;
```
Comment (bseptember) [inline src/app/components/hris/sign-in/sign-in.component.ts:25]: Refactor: this line I don't think is necessary, the variable is not being used.

File: src/app/components/hris/employees/employee-approvals/view-starter-kit-approval/view-starter-kit-approval.component.html [hunk 3]
```diff
@@ -19,8 +19,8 @@
         <div id="approval-header" class="row p-2">
             <div class="col-2 mt-3 mb-3">
                 <div id="profile-image">
-                    <img [src]="employee?.photo == '' || employee?.photo == 'TBA' ? 
-                        '../../../../../assets/img/default-profile-image.png' : employee?.photo"
+                    <img [src]="authAccessService.getAuthTokenProfilePicture() || '../../../../../../assets/img/default-profile-image.png'"
+          alt="Profile img" class="rounded-circle profile-img">
```
Comment (bseptember) [inline src/app/components/hris/employees/employee-approvals/view-starter-kit-approval/view-starter-kit-approval.component.html:23]: Fix: the logic here should be the same as in the others -

1) first use employee profile photo, check if available
2) if employee profile photo not available, use google photo
3) if google photo not available use static image

File: src/app/components/hris/employees/employee-approvals/view-banking-approval/view-banking-approval.component.html [hunk 4]
```diff
@@ -18,7 +18,8 @@
         <div id="approval-header" class="row p-2">
             <div class="col-2 mt-3 mb-3">
                 <div id="profile-image">
-                    <img [src]="employee?.photo == '' ? '../../../../../assets/img/default-profile-image.png' : employee?.photo" alt="">
+                    <img [src]="this.authAccessService.getAuthTokenProfilePicture() || '../../../../../../assets/img/default-profile-image.png'"
+          alt="Profile img" class="rounded-circle profile-img">>
```
Comment (bseptember) [inline src/app/components/hris/employees/employee-approvals/view-banking-approval/view-banking-approval.component.html:22]: Fix: the logic here should be the same as in the others -

first use employee.photo, check if available
if employee profile photo not available, use google photo
if google photo not available use static image

**Review by bseptember — APPROVED** [review]

**Review by ReinoRabbit — APPROVED** [review]

**Review by Wimmas — APPROVED** [review]

**Pull request merged**